### ISSUE TITLE
[Quests] Active Item Quests

### ISSUE THREAD

**Issue by EnlightenedEmperor** [Valhalla-Gaming/Tracker#8557]
**Describe the issue you're having**: Vast majority of quests which require you to use an item (The kind of quests where quest tracker hovers a clickable item icon next to the name and counter) on a certain npc / mob / area are bugged - doing an action doesn't increase the 0/x counter.

**Explain how you expect it work**: Self-explanatory.

**Steps to reproduce the problem**: Attempt pretty much any quest of the kind.

**Comment by Analog-Valhalla** [comment]
Hello there!
If this issue is still happening, could you provide more information about it? Perhaps some screenshots or examples as well! Thanks :smile:

**Comment by EnlightenedEmperor** [comment]
I'll record it the next time i'll encounter this kind of quest, i think that a video would be the best way to show it.

**Comment by malex1024** [comment]
> I'll record it the next time i'll encounter this kind of quest, i think that a video would be the best way to show it.

No, we need links. Feel free to provide both, a visual of the bugged behavior is never a bad thing - but links. We always need links.

Thanks for the bug report, though! 👍
